app/controllers/reports/regions_controller.rb

```diff
@@ -50,24 +50,19 @@ def details
 
   def cohort
     authorize(:dashboard, :show?)
+    period_params = report_params[:period].presence || {type: :month, value: Date.current.beginning_of_month}
```

Comment:
`report_params[:period].presence || CohortService.default_period`?

Reply:
Agreed, let's delegate this default to the service as we're doing in `set_period`